sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -273,45 +304,38 @@ def _to_outgoing_amqp_message(self):
                 absolute_expiry_time=absolute_expiry_time,
                 group_id=self.properties.group_id,
                 group_sequence=self.properties.group_sequence,
-                reply_to_group_id=self.properties.reply_to_group_id,
-                encoding=self._encoding
+                reply_to_group_id=self.properties.reply_to_group_id
             )
         elif ttl_set:
-            message_properties = uamqp.message.MessageProperties(
+            message_properties = Properties(
                 creation_time=creation_time_from_ttl if ttl_set else None,
                 absolute_expiry_time=absolute_expiry_time_from_ttl if ttl_set else None,
             )
-
-        amqp_body = self._message._body  # pylint: disable=protected-access
-        if isinstance(amqp_body, uamqp.message.DataBody):
-            amqp_body_type = uamqp.MessageBodyType.Data
-            amqp_body = list(amqp_body.data)
-        elif isinstance(amqp_body, uamqp.message.SequenceBody):
-            amqp_body_type = uamqp.MessageBodyType.Sequence
-            amqp_body = list(amqp_body.data)
-        else:
-            # amqp_body is type of uamqp.message.ValueBody
-            amqp_body_type = uamqp.MessageBodyType.Value
-            amqp_body = amqp_body.data
-
-        return uamqp.message.Message(
-            body=amqp_body,
-            body_type=amqp_body_type,
+        annotations = None
+        if self.annotations:
+            # TODO: Investigate how we originally encoded annotations.
+            annotations = dict(self.annotations)
+            for key in _LONG_ANNOTATIONS:
+                if key in self.annotations:
+                    annotations[key] = amqp_long_value(self.annotations[key])
```

Comment:
are _LONG_ANNOTATIONS only relevant for SB? I don't see these in EH, so am currently passing in `message_annotations=self.annotations` to the pyamqp Message there.